### DISCUSSION TITLE
Addon na Stremio

### DISCUSSION THREAD

**Discussion by youchi1** [Seraphim-Solutions/bettercinema#31] [Q&A] [answered]
Ahoj, neuvažovali ste o addone do stremio.com? Stremio je FOSS appka s asi najkrajším a najprívetivejším UI.

Dik za odpoveď.

**Comment by DanniSec** [comment, marked as answer]
Ahoj, o stremio víme, ale na to momentálně už existuje [extension](https://webshare-stremio.vercel.app/) který tohle již implementuje.

Momentálně jsem se po delší době vrátil k developmentu BetterCinema s tím že je tam hodně věcí co bych chtěl předělat atd.

Cílem tohohle projektu bylo udelělat CLI interface s možným GUI v budoucnu (včetně aplikace pro smart TV atd.) ale na to zatím nebyl čas.

Přidal bych stremio jako možný přehrávač aspoň ale to pokud vím stremio samo zatím neumožňuje.

Ale přesto děkuji za projevený zájem a otázku ;)

**Reply by youchi1** [reply]
Super, diki moc za odpoveď a link na addon, funguje super. Vieš ma nasmerovať, kde by som sa mohol dozvedieť viac o tomto addone, prípadne vývoj možno aj podporiť? Žiadny vyhľadávač ho nepozná, ani pri exact searchovaní samotného linku :)

Držím palce s plánmi :)

**Reply by DanniSec** [reply]
Ahoj, stremio extension už je taky na githubu https://github.com/ljsydpwym/StremioCinema.

**Comment by trivarialthea** [comment]
> Super, diki moc za odpoveď a link na addon, funguje super. Vieš ma nasmerovať, kde by som sa mohol dozvedieť viac o tomto addone, prípadne vývoj možno aj podporiť? Žiadny vyhľadávač ho nepozná, ani pri exact searchovaní samotného linku :)
> 
> Držím palce s plánmi :)

Ahoj, 
tvůrcem addonu je Dominik, na Discordu ho najdeš jako "micutad" asi ti bude schopný odpovědět nejlépe.
